pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/VenueImageDeleteModal.tsx

```diff
@@ -1,24 +1,42 @@
-import React, { FunctionComponent } from 'react'
+import React, { useCallback, useState } from 'react'
 
-import DialogBox from 'new_components/DialogBox'
+import { ReactComponent as TrashIcon } from 'icons/ico-trash.svg'
+import ConfirmDialog from 'new_components/ConfirmDialog'
+import { deleteVenueImage } from 'repository/pcapi/pcapi'
 
-import { VenueImageDelete } from '../VenueImageDelete/VenueImageDelete'
-
-type Props = {
+interface IVenueImageDeleteModalProps {
+  venueId: string
+  onDeleteImage: () => void
   onDismiss: () => void
   children?: never
 }
 
-export const VenueImageDeleteModal: FunctionComponent<Props> = ({
+export const VenueImageDeleteModal = ({
+  venueId,
+  onDeleteImage,
   onDismiss,
-}) => {
+}: IVenueImageDeleteModalProps): JSX.Element => {
+  const [isDeleting, setIsDeleting] = useState(false)
+  const tryToDeleteVenueImage = useCallback(async () => {
```

Comment:
```suggestion
  const deleteVenueImage = useCallback(async () => {
```
ca ira tres bien :)
En plus en l'état on ne gère pas le retour en erreur

Reply:
`onDeleteImage` est déjà utilisé pour la fonction qui appelle l'API